src/cmd/promremotebench/checker.go

```diff
@@ -104,6 +119,44 @@ func (c *checker) GetHostNames() []string {
 	return results
 }
 
+func (c *checker) cleanupLoop(loopDuration time.Duration) {
+	hostsToRemove := make([]string, 0, 4)
```

Comment:
Want to make the capacity configurable?

Reply:
Originally figured to just let this grow over time since we use the same slice again and again, but I'll plumb through numHosts from main.